docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+services:
+  minio:
+    container_name: minio_container
+    network_mode: host
+    image: quay.io/minio/minio
+    command: server /data --console-address ":9001"
+    environment:
+      - "MINIO_ROOT_USER=AKIAIOSFODNN7EXAMPLE"
+      - "MINIO_ROOT_PASSWORD=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+  miniocreatebuckets:
+    image: minio/mc
+    network_mode: host
+    depends_on:
+      - minio
+    entrypoint: >
+      /bin/sh -c "
+      until (mc alias set hummock-minio http://127.0.0.1:9000 AKIAIOSFODNN7EXAMPLE wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY) do echo '...waiting...' && sleep 1; done;
+      mc admin user add hummock-minio/ hummock 12345678;
+      mc admin policy set hummock-minio/ readwrite user=hummock;
+      mc mb hummock-minio/hummock001;
+      exit 0;
+      "
+  metanode:
+    container_name: metanode_container
+    network_mode: host
+    image: ghcr.io/singularity-data/risingwave-backend:latest
+    command: /bin/bash -c /risingwave/bin/meta-node --host 127.0.0.1:5690 --dashboard-host 127.0.0.1:5691 --prometheus-host 127.0.0.1:1250
+    depends_on:
+      - miniocreatebuckets
```

Comment:
It actually do not depend on miniocreatebuckets. It can be execute with minio&miniocreatebuckets in parallel.